### PR TITLE
Release 0.1.0

### PR DESCRIPTION
**Pull request by better0fdead**
## Overview

The release introduces the `metrics-export-role` module: export metrics from `Tarantool` 3.
The release contains role for `Tarantool` 3.

## New features

* `Tarantool` 3 role with documentation.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,10 +9,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
-
 ### Fixed
 
 ### Changed
 
+## 0.1.0 - 2024-06-11
+
+The release introduces the `metrics-export-role` module: export metrics
+from `Tarantool` 3.
+The release contains role for `Tarantool` 3 with documentation.
```
Comment (oleg-jukovec) [inline CHANGELOG.md]:
```suggestion
from `Tarantool` 3. The release contains role for `Tarantool` 3 with
documentation.
```

**Review by oleg-jukovec — APPROVED** [review]
LGTM!

File: .github/workflows/publish.yml [hunk 2]
```diff
@@ -14,7 +14,7 @@ jobs:
       - name: Check module version
         uses: tarantool/actions/check-module-version@master
         with:
-          module-name: 'metrics-export-role'
+          module-name: 'roles.metrics-export'
```
Comment (DifferentialOrange) [inline .github/workflows/publish.yml]: Is it works fine? As far as I remember, action uses Tarantool 2.10

**Review by oleg-jukovec — COMMENTED** [review]
```
Wrote our own version-check because 'tarantool/check-module-version' action installs Tarantool 2.10.
For this module we need Tarantool 3.
```

Please, update the commit message according the rules (72 characters per a line):

https://www.tarantool.io/en/doc/latest/dev_guide/developer_guidelines/#how-to-write-a-commit-message

File: .github/workflows/publish.yml (hunk 2 above)
Comment (better0fdead) [reply .github/workflows/publish.yml]: Thx, changed.
For now reworked this job.
Will open PR in tarantool/actions to add an ability to choose tarantool version.

**Comment by better0fdead** [comment]
> ```
> Wrote our own version-check because 'tarantool/check-module-version' action installs Tarantool 2.10.
> For this module we need Tarantool 3.
> ```
> 
> Please, update the commit message according the rules (72 characters per a line):
> 
> https://www.tarantool.io/en/doc/latest/dev_guide/developer_guidelines/#how-to-write-a-commit-message

thx, changed

**Review by DifferentialOrange — APPROVED** [review]

**Pull request merged**